Makefile

```diff
@@ -11,6 +11,7 @@ GOLANG_BASE	= presidio-golang-base
 GIT_TAG   = $(shell git describe --tags --always 2>/dev/null)
 VERSION   ?= ${GIT_TAG}
 PRESIDIO_LABEL := $(if $(PRESIDIO_LABEL),$(PRESIDIO_LABEL),$(VERSION))
+RELEASE_VERSION := 
```

Comment:
where is used?

Reply:
below in the push release section

Reply:
ok. why is it empty?

Reply:
same as 'DOCKER_BUILD_FLAGS', it is easy to see that this variable exists/needed when you declare it explicitly. but other than that no specific purpose. currently it is populated when passing it with the make command